backend/src/main/java/com/redhat/cloud/notifications/processors/webhooks/WebhookTypeProcessor.java

```diff
@@ -117,26 +117,26 @@ private WebClient getWebClient(boolean disableSSLVerification) {
                         .onItem().transform(resp -> {
                             NotificationHistory history = buildNotificationHistory(item, startTime);
 
+                            HttpRequestImpl<Buffer> reqImpl = (HttpRequestImpl<Buffer>) req.getDelegate();
+
                             boolean serverError = false;
                             if (resp.statusCode() >= 200 && resp.statusCode() < 300) {
                                 // Accepted
-                                LOGGER.debugf("Target endpoint successful: %d", resp.statusCode());
+                                LOGGER.debugf("Webhook request to %s was successful: %d", reqImpl.host(), resp.statusCode());
                                 history.setInvocationResult(true);
                             } else if (resp.statusCode() >= 500) {
                                 // Temporary error, allow retry
                                 serverError = true;
-                                LOGGER.debugf("Target endpoint server error: %d %s", resp.statusCode(), resp.statusMessage());
+                                LOGGER.debugf("Webhook request to %s failed: %d %s", reqImpl.host(), resp.statusCode(), resp.statusMessage());
```

Comment:
Should we say that we will retry?

Reply:
Oops I saw this after clicking on merge, sorry 😄 That would have made sense, yeah, but now we'll have to wait for another PR.